minkindr_python/src/export-rotation-quaternion.cc

```diff
@@ -0,0 +1,70 @@
+#include <glog/logging.h>
+#include <kindr/minimal/rotation-quaternion.h>
+#include <numpy_eigen/boost_python_headers.hpp>
+
+using namespace boost::python;
+
+typedef kindr::minimal::RotationQuaternionTemplate<double> Quaternion;
+
+Eigen::Vector4d getQuaternionXYZW(const Quaternion* quaternion) {
+  return CHECK_NOTNULL(quaternion)->toImplementation().coeffs();
```

Comment:
I overlooked this before, but I would not rely on internal ordering if possible. It is  not even specified in the Eigen doc (at least not prominently). Hence, I would just copy from below (the usage of w() ..) and reorder for safety. 

Btw: you should be able to use const ref for the quaternion and safe the NOTNULL check (same below).